monai/bundle/properties.py

```diff
@@ -216,3 +221,42 @@
         BundlePropertyConfig.REF_ID: f"evaluator{ID_SEP_KEY}key_val_metric",
     },
 }
+
+MetaProterties = {
+    "version": {
+        BundleProperty.DESC: "bundle version",
+        BundleProperty.REQUIRED: False,
+        BundlePropertyConfig.ID: f"_meta_{ID_SEP_KEY}version",
+    },
+    "monai_version": {
+        BundleProperty.DESC: "required monai version used for bundle",
+        BundleProperty.REQUIRED: False,
```

Comment:
MONAI version may be required?

Reply:
Hi @Nic-Ma, I don't have these two properties set to required given these issues. Not sure whether I need to do that.
https://github.com/Project-MONAI/tutorials/issues/1455
https://github.com/Project-MONAI/tutorials/issues/1454
https://github.com/Project-MONAI/tutorials/issues/1453
https://github.com/Project-MONAI/tutorials/issues/1452